sql/install.sql

```diff
@@ -1,5 +1,18 @@
-CREATE TABLE `PREFIX_pmt_cart_process` (
+CREATE TABLE IF NOT EXISTS `PREFIX_pmt_cart_process` (
   `id` INT NOT NULL ,
   `timestamp` INT NOT NULL ,
   PRIMARY KEY (`id`)
   ) ENGINE = InnoDB;
+
+CREATE TABLE IF NOT EXISTS `PREFIX_pmt_order` (
```

Comment:
A partir de que versión de MySQL se puede usar el IF NOT EXISTS?

ten en cuenta que puede ser un mysql muy antiguo

Reply:
PHP 5.3 released at 30 June 2009
The first release (not beta) of mysql was Version 5.1 at November 2008 (next was 5.5 in 2010) so the function must exists in 5.q version...checking

Reply:
Is supported.
http://ftp.nchu.edu.tw/MySQL/doc/refman/5.1/en/create-table.html